src/lib/serviceRoutinesV2/postBatchQuery.cpp

```diff
@@ -51,6 +53,7 @@
 *   - limit=NUMBER
 *   - offset=NUMBER
 *   - options=count,keyValues
+*   - q
```

Comment:
"q" as URL param is only valid in "GET /v2/entities" operation (please check http://telefonicaid.github.io/fiware-orion/api/v2/)